gulpfile.js

```diff
@@ -8,7 +8,7 @@ gulp.task('serve', function(){
         ui: { port: 8081 }
     };
     browserSync.init(options);
-    gulp.watch('public/**/*.*').on('change', browserSync.reload);
+    gulp.watch('public/*.*/*.*').on('change', browserSync.reload);
```

Comment:
You should remove this dot. the /**/*.* means that it will also check sub directories for any file changes. If you leave it as you have it now you will have issues with gulp.